File-Formats/DocIO/Working-with-Table-Of-Contents.md

```diff
@@ -581,10 +581,358 @@ document.Close();
 //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
 {% endhighlight %}
 
-{% endtabs %}  
+{% endtabs %}
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Table-Of-Contents/TOC-with-user-defined-styles).
 
+## Applying switches for table of contents
+
+Switches are used to change the default settings in word document. TOC will determine the TOC entries based on the TOC switches. You can set heading levels, specify whether to use default heading styles, show page numbers, set page numbers to right alignment, set hyperlinks for TOC levels, preserve line break and text wrapping break in TOC, set outline levels for paragraph and specify whether to use table of entry fields. 
+
+The following code example illustrates how to apply switches for table of contents.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Create a new Word document.
+using (WordDocument document = new WordDocument())
+{
+    //Add a section into the Word document.
+    IWSection section = document.AddSection();
+    //Add a paragraph into the created section.
+    IWParagraph paragraph = section.AddParagraph();
+    // Append table of content to the end of the paragraph with the specified lower and upper heading levels.
```

Comment:
//Appends the TOC field with LowerHeadingLevel and UpperHeadingLevel to determines the TOC entries